presto-kafka/src/test/java/io/prestosql/plugin/kafka/KafkaQueryRunner.java

```diff
@@ -160,16 +164,19 @@ private static DistributedQueryRunner createKafkaQueryRunner(
                     .build();
             KafkaPlugin kafkaPlugin = new KafkaPlugin(combine(
                     extensions,
-                    binder -> newSetBinder(binder, TableDescriptionSupplier.class)
-                            .addBinding()
-                            .toInstance(new MapBasedTableDescriptionSupplier(topicDescriptions))));
+                    installModuleIf(
+                            KafkaConfig.class,
+                            kafkaConfig -> kafkaConfig.getTopicSchemaProvider().equalsIgnoreCase(TEST),
+                            binder -> newMapBinder(binder, String.class, TopicSchemaProvider.class)
+                                    .addBinding(TEST)
+                                    .toInstance(new MapBasedTopicSchemaProvider(topicDescriptions)))));
             queryRunner.installPlugin(kafkaPlugin);
 
             Map<String, String> kafkaProperties = new HashMap<>(ImmutableMap.copyOf(extraKafkaProperties));
             kafkaProperties.putIfAbsent("kafka.nodes", testingKafka.getConnectString());
+            kafkaProperties.putIfAbsent("kafka.table-description-supplier", TEST);
             kafkaProperties.putIfAbsent("kafka.default-schema", "default");
             kafkaProperties.putIfAbsent("kafka.messages-per-split", "1000");
-            kafkaProperties.putIfAbsent("kafka.table-description-dir", "write-test");
```

Comment:
Was this needed by tests?

Reply:
No, the tests passed without it locally.